include/sensors/core/tasks/capacitive_driver.hpp

```diff
@@ -38,19 +44,27 @@ class FDC1004 {
     [[nodiscard]] auto initialized() const -> bool { return _initialized; }
 
     auto initialize() -> void {
-        // FIXME we should grab device id and compare it to the static
-        // device id in code.
-
-        // We should send a message that the sensor is in a ready state,
-        // not sure if we should have a separate can message to do that
-        // holding off for this PR.
-        if (shared_sensor) {
-            measure_mode = fdc1004::MeasureConfigMode::TWO;
+        if (!_initialized) {
+            // FIXME we should grab device id and compare it to the static
+            // device id in code.
+
+            // We should send a message that the sensor is in a ready state,
```

Comment:
we'd want a query-response cycle for this rather than just a notification so we can handle if the pipette was up before the server as is common when you boot the machine with the pipettes on it.